md5/src/block_api.rs

```diff
@@ -10,7 +10,8 @@ use digest::{
     typenum::{U16, U24, U64, Unsigned},
 };
 
-use crate::{compress::compress, consts};
+pub use crate::compress::compress;
+use crate::consts;
```

Comment:
Changed to export `md5::block_api::compress` here like in `sha2`

Reply:
I think it's better to do it in a separate PR. Do not forget to add a changelog entry.

Reply:
done.